pkg/iac/ignore/parse.go

```diff
@@ -5,6 +5,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/samber/lo"
```

Comment:
Unrelated to your change (GitHub won't allow me to comment on the unchanged line) but should we check if `len(rules)>0` [here](https://github.com/aquasecurity/trivy/pull/6708/files#diff-c1fe13f24e97f852991501a2535948a2ff13ef4e36564313e85754d8f9c6f35eR33) or we risk another panic.

Reply:
Why would a zero slice cause panic?

Reply:
ah right - I saw it in my IDE but didn't realize it was a false positive.